Brightspace Extensions/BrightspaceExtensions.postman_collection.json

```diff
@@ -0,0 +1,385 @@
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{BrightspaceDomain}}/d2l/api/BrightspaceExtensions/extensions/{{Extension1}}",
+							"host": [
+								"{{BrightspaceDomain}}"
+							],
+							"path": [
+								"d2l",
+								"api",
+								"BrightspaceExtensions",
+								"extensions",
+								"{{Extension1}}"
+							]
+						},
+						"description": "Update Extension 1's Name"
+					},
+					"response": []
+				},
+				{
+					"name": "Get Extension 1",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var jsonData = JSON.parse(responseBody);\r",
+									"\r",
+									"pm.test(\"Checking that extension one's name has been changed\", function() { jsonData.Name == \"THIS HAS BEEN CHANGED\" })\r",
+									"\r",
+									"pm.test(\"Response should have 200 Code\", function() {pm.response.to.have.status(200);})"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{BrightspaceDomain}}/d2l/api/BrightspaceExtensions/extensions/{{Extension1}}",
+							"host": [
+								"{{BrightspaceDomain}}"
+							],
+							"path": [
+								"d2l",
+								"api",
+								"BrightspaceExtensions",
+								"extensions",
+								"{{Extension1}}"
+							]
+						},
+						"description": "Ensure Extension 1's name has changed as expected"
+					},
+					"response": []
+				},
+				{
+					"name": "Get All Extensions",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"var jsonData = JSON.parse(responseBody);\r",
+									"var initialExtnesionCount = pm.collectionVariables.get(\"ExtensionNumber\");\r",
+									"pm.test(\"Checking that extension one's name has been changed\", function() { jsonData.length == initialExtnesionCount+2 })\r",
+									"pm.test(\"Response should have 200 Code\", function() {pm.response.to.have.status(200);})"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{BrightspaceDomain}}/d2l/api/BrightspaceExtensions/extensions",
+							"host": [
+								"{{BrightspaceDomain}}"
+							],
+							"path": [
+								"d2l",
+								"api",
+								"BrightspaceExtensions",
+								"extensions"
+							]
+						},
+						"description": "Checks we only made 2 users during these postman calls (usuing initial extension count)"
```

Comment:
Nit: should the description be `Checks we only made 2 extensions...` instead of `2 users`?  And spelling nit `usuing` => `using`.